common/api/ui-framework.api.md

```diff
@@ -6569,12 +6569,22 @@ export interface VisibilityComponentProps {
     iModelConnection: IModelConnection;
 }
 
+// @alpha
+export enum VisibilityState {
+    // (undocumented)
+    Hidden = 2,
+    // (undocumented)
+    PartiallyVisible = 1,
+    // (undocumented)
+    Visible = 0
+}
+
 // @alpha
 export interface VisibilityStatus {
     // (undocumented)
     isDisabled?: boolean;
     // (undocumented)
-    isDisplayed: boolean;
+    state: VisibilityState;

```

Comment:
I hate seeing `VisibilityStatus` and `VisibilityState` being used next to each other - very hard to distinguish what is what. Any ideas for other naming? If not, I'd even go as far as changing `state: VisibilityState` to `displayState: "hidden" | "partial" | "visible"`